webrender/src/device/gl.rs

```diff
@@ -2,47 +2,38 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
let's do these file moves in WR upstream first, and not rush with this here in the mean time to avoid diverging code histories